products/products.model.ts

```diff
@@ -0,0 +1,16 @@
+
+export class Product{
+    // id:string;
+    // title:string;
+    // description:string;
+    // price:number;
+    constructor(public id:string,
+        public title:string,
+        public description:string,
+        public price:number)
```

Comment:
why you used constructor while writing the schema.?
as we have store the data in the form of json so create the interface of product object.